### PR TITLE
RavenDB-4840 studio: Add an option to view the list of active huge do…

### PR DESCRIPTION
**Pull request by ml054**
…cuments

### CODE REVIEW

File: src/Raven.Server/Documents/HugeDocuments.cs [hunk 1]
```diff
@@ -1,35 +1,95 @@
...
+using Raven.Server.NotificationCenter.Notifications;
+using Raven.Server.NotificationCenter.Notifications.Details;
 using Raven.Server.Utils;
+using Sparrow.Json;
 
 namespace Raven.Server.Documents
 {
     public class HugeDocuments
     {
+        private static readonly string PerformanceHintSource = "Documents";
```
Comment (arekpalinski) [inline src/Raven.Server/Documents/HugeDocuments.cs:16]: Documents -> HugeDocuments

File: src/Raven.Server/Documents/HugeDocuments.cs [hunk 2]
```diff
@@ -1,35 +1,95 @@
...
+                        typeof(HugeDocumentsDetails),
+                        HugeDocumentsId,
+                        detailsJson,
+                        DocumentConventions.Default);
+                }
+
+                return PerformanceHint.Create(
+                    _database,
+                    "Huge documents",
+                    "We have detected that some documents has surpassed the configured size threshold",
```
Comment (arekpalinski) [inline src/Raven.Server/Documents/HugeDocuments.cs]: We need a better message here:
- explain that huge docs might impact performance
- add the current threshold
- provide configuration key name

**Review by arekpalinski — CHANGES_REQUESTED** [review]
Please add a test

File: src/Raven.Server/Documents/HugeDocuments.cs (hunk 1 above)
Comment (ml054) [reply src/Raven.Server/Documents/HugeDocuments.cs:16]: talked with @arekpalinski - ignoring

File: src/Raven.Server/Documents/HugeDocuments.cs (hunk 2 above)
Comment (ml054) [reply src/Raven.Server/Documents/HugeDocuments.cs]: updated.

**Comment by ppekrol** [comment]
Rebased. Please add a test.

**Comment by ml054** [comment]
test is now added. Please verify.

**Review by arekpalinski — APPROVED** [review]

**Comment by ppekrol** [comment]
Can you check the failing test?

**Comment by ml054** [comment]
fixed.

**Pull request merged**